README.md

```diff
@@ -3,8 +3,54 @@
 This a pure Scala implementation of [Babelfish](https://doc.bblf.sh/) client.
 It uses [ScalaPB](https://scalapb.github.io/grpc.html) for Protobuf/gRPC code generation.
 
+### Installation
+
+```
+./sbt assembly
+```
+
+gRPC/protobuf are re-generate from `src/main/proto` on every `./sbt compile` and
+are stored under `./target/scala-2.11/src_managed/`. 
+
+The jar file and the native module are generated in the `build/` directory. If 
+you move the jar file to some other path, the native (`.so` or `.dylib`) 
+library must be in the same path.
+
+If the build fails because it can find the `jni.h` header file, run it with:
+
+```
+./sbt -java_path /usr/lib/jvm/java-8-openjdk-amd64 assembly
+```
+
+Changing the JDK directory to the one right for your system.
+
+*Note: currently only Linux is supported, check [this
```

Comment:
`[this issue](https://github.com/bblfsh/client-scala/issues/7)` could just be replaced by `#7` ;)